src/t_stream.c

```diff
@@ -1370,24 +1370,29 @@ void streamLastValidID(stream *s, streamID *maxid)
     streamIteratorStop(&si);
 }
 
+#define STREAM_ID_STR_LEN 44 /* At least 20 * 2 + 1 */
```

Comment:
41?

Reply:
i'm paranoid about off-by one errors leading to security issues.
i would go with a round 48, but then when sds adds 4 more bytes (header and null term), it'll use a bigger bin size.
so 44 gives us a few bytes for safety, and the sds still remains in the 48 bin.
should i document all that in the code, so that people won't come up with PRs to fix it?
note that no one tried to reduce MAX_LONG_DOUBLE_CHARS from 5k to 4952

Reply:
Without your explanation, I would have thought it was a typo mistake, but I can't imagine it was for security reasons since `sdscatfmt` can automatically expand.

Reply:
yeah, when creating such a define, i was paranoid about how it was going to be used in the future. i'll add a comment.